From 1f6b8cf689043c433438aaca9eeebb22c1494fbd Mon Sep 17 00:00:00 2001
From: RemiliaForever <root@koumakan-ryzen>
Date: Wed, 2 Dec 2020 17:50:04 +0800
Subject: [PATCH] diff

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 3cd617a..e801c7a 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -577,9 +577,9 @@ if(CUDA_FOUND)
             if (CUDA_VERSION_MAJOR LESS 6)
                 set( OSD_CUDA_NVCC_FLAGS --gpu-architecture compute_11 )
             elseif (CUDA_VERSION_MAJOR LESS 9)
-                set( OSD_CUDA_NVCC_FLAGS --gpu-architecture compute_20 )
-            else()
                 set( OSD_CUDA_NVCC_FLAGS --gpu-architecture compute_30 )
+            else()
+                set( OSD_CUDA_NVCC_FLAGS --gpu-architecture compute_52 )
             endif()
         endif()
     endif()
-- 
2.26.2

